php/class-fm-bylines.php

```diff
@@ -269,6 +307,28 @@ function add_byline_meta_box( $type = 'author', $label = null, $args = array() )
 		}
 
 		/**
+		 * Only set the default for the first element shown.
+		 */
+		public function display_default_byline( $classes, $name, $field ) {
+			if ( ! empty( $field->data_id ) && get_post_type( $field->data_id ) != $this->name ) {
+				$pattern = '/^fm-fm_bylines_(.*)-(.*)-.*-\d$/';
+				preg_match( $pattern, $field->get_element_id(), $matches );
+				if ( 'byline_id' == $name && ! empty( $matches ) ) {
+					$type = $matches[1];
+					$index = $matches[2];
+					// On new posts only, use default values.  This allows for empty bylines.
+					if ( absint( $index ) === 0 && get_post_status( $field->data_id ) === 'auto-draft' ) {
+						$field->default_value = ( apply_filters( "fm_bylines_{$type}_enable_user_mapping_defaults", $this->get_byline_user_mapping() ) ) ? $this->get_byline_user_mapping() : null;
+					} else {
+						$field->default_value = null;
```

Comment:
It populates every meta box. The hooks tells it to populate every metabox.  Once you overwrite it for the auto draft, I wanted to make sure it went back to its original state.